prometheus/exporter.go

```diff
@@ -40,6 +40,9 @@ type Exporter struct {
 
 	// Consumer Groups
 	consumerGroupInfo              *prometheus.Desc
+	consumerGroupMembersEmpty      *prometheus.Desc
+	consumerGroupTopicMembers      *prometheus.Desc
+	consumerGroupTopicPartitions   *prometheus.Desc
```

Comment:
```suggestion
	consumerGroupAssignedTopicPartitions   *prometheus.Desc
```

Just for clarification because otherwise I'd expect the number of partitions where this consumer group has active offsets for.

Reply:
👍🏻